pykern/pkcli/test.py

```diff
@@ -18,8 +18,9 @@
 
 SUITE_D = "tests"
 _COROUTINE_NEVER_AWAITED = re.compile("RuntimeWarning: coroutine .+ was never awaited")
+_FAILED_ON_WARNINGS = "\nFAILED due to:\n"
```

Comment:
It's very specific so make the name specific. This doesn't need to be a constant.